src/components/AuthSection/AuthComponent.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react';
+
+// Components
+import { LoginCard, ProfileCard } from '.';
+
+const AuthComponent = () => {
+  const [login] = useState(false);
+  return <div>{login ? <ProfileCard /> : <LoginCard />}</div>;
```

Comment:
Do we need an empty `<div>` here?
https://reactjs.org/docs/fragments.html#short-syntax